packages/spark-core/setup-core.scss

```diff
@@ -0,0 +1,62 @@
+///
```

Comment:
Should we have 3 files instead of 1? Maybe `_scales.scss`, `_spark-core.scss`, and `spark.scss`.

Then the main import file only needs a couple lines? Is that more or less confusing?